apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.Data/ErrorCode.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Grpc.Core;
+
+namespace Google.Cloud.Spanner
```

Comment:
The namespace here doesn't match the location. We should talk about version-neutral vs versioned namespaces. (It would be odd to have a version-neutral namespace in a versioned assembly, IMO. There's lots to talk about in terms of how we ship in general though.)